packages/cli/test/RuntimeTest.ts

```diff
@@ -22,26 +22,30 @@
 
 import { suite, test } from "@testdeck/mocha";
 import { should, assert } from "chai";
+import DefaultServient from "../src/cli-default-servient";
 
-import Servient from "../src/servient";
 import fs from "fs";
 // should must be called to augment all variables
 should();
 
-@suite("the runtime of servient")
+@suite("Test suite for script runtime")
 class WoTRuntimeTest {
-    static servient: Servient;
+    static servient: DefaultServient;
 
     static WoT: typeof WoT;
 
     exit: (code?: number) => never;
 
-    static before() {
+    static async before() {
         // We need to disable this check for killing servient logs
-        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        /* eslint-disable @typescript-eslint/no-empty-function */
         console.error = () => {};
-        this.servient = new Servient();
-        console.log("before starting test suite");
+        console.debug = () => {};
+        console.warn = () => {};
+        console.info = () => {};
+        /* eslint-enable @typescript-eslint/no-empty-function */
```

Comment:
Maybe not, since it just uses another comment style */ vs //

Reply:
We have two choices:
- `console.debug = () => { /* nop */ }`
- `/* eslint-disable @typescript-eslint/no-empty-function */` with the corresponding `/* eslint-enable @typescript-eslint/no-empty-function */` 

Personally, I found the first a workaround as it does not clearly explain why we are adding a comment in an empty function.

Reply:
we can also change a little bit [eslint configuration](https://eslint.org/docs/rules/no-empty-function#:~:text=Rule%20Details,if%20it%20contains%20a%20comment.&text=Examples%20of%20correct%20code%20for,)%20%7B%20%2F%2F%20do%20nothing.%20%7D) to allow empty arrow functions.

Reply:
I am fine as is. I was just wondering.